tests/minimized/src/main/java/minimized/Methods.java

```diff
@@ -17,7 +17,7 @@ private static String staticOverload(String value) {
     return value + "1";
   }
 
-  public static String app(int n, String m) {
+  public static String app(int n, String m) throws RuntimeException {
```

Comment:
I'm surprised that epoxy doesn't use `throws` anywhere. Checked exceptions are IMO one of the biggest pains of working with Java. Can we add one test case where the exception has type parameters just to stress the code path that `throws` is `Type` and not symbols

Reply:
Turns out custom exceptions (that extend Exception/Throwable) cant have type parameters :thinking:

Reply:
Where did you find examples with type parameters? Are they only in the JDK?

Reply:
https://sourcegraph.com/search?q=context:global+%28+throws+%5Ba-zA-Z%5D%2B%3C%5BA-Z%5D%5Ba-zA-Z%5D*%3E%29+lang:java&patternType=regexp&case=yes

Reply:
Not exactly "conclusive" or "proving" evidence by any means :grin:

Reply:
Ive added an additional exception to the signature to make up for it. Gonna merge :+1: